pkg/handler/enqueue_owner.go

```diff
@@ -126,10 +130,21 @@ func (e *EnqueueRequestForOwner) getOwnerReconcileRequest(object metav1.Object)
 		// object in the event.
 		if ref.Kind == e.groupKind.Kind && refGV.Group == e.groupKind.Group {
 			// Match found - add a Request for the object referred to in the OwnerReference
-			result = append(result, reconcile.Request{NamespacedName: types.NamespacedName{
-				Namespace: object.GetNamespace(),
-				Name:      ref.Name,
-			}})
+			r := reconcile.Request{NamespacedName: types.NamespacedName{
+				Name: ref.Name,
+			}}
+
+			// if owner is not namespaced then we should set the namespace to the empty
+			mapping, err := e.mapper.RESTMapping(e.groupKind, refGV.Version)
+			if err != nil {
+				log.Error(err, "Could not retrieve rest mapping", "group", e.groupKind.Group, "kindv", e.groupKind.Kind)
```

Comment:
"kindv"?

Also: https://github.com/kubernetes-sigs/controller-runtime/blob/master/TMP-LOGGING.md#groups-versions-and-kinds (just use the "kind" tag with the groupKind directly)